po/fr.po

```diff
@@ -28346,122 +28408,119 @@ msgstr ""
 
 #: /tmp/fish/implicit/share/completions/grep.fish:2
 msgid "Process binary file as text"
-msgstr ""
+msgstr "Traiter les fichiers binaires comme du texte"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:5
 msgid "Print byte offset of matches"
-msgstr ""
+msgstr "Afficher le décalage en octets des occurrences"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:6
 msgid "Assume data type for binary files"
-msgstr ""
+msgstr "Présumer du type de données pour les fichiers binaires"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:7
 msgid "Colour output"
-msgstr ""
+msgstr "Colorer la sortie"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:8
 #: /tmp/fish/implicit/share/completions/jq.fish:10
 msgid "Color output"
-msgstr ""
+msgstr "Colorer la sortie"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:9
 msgid "Only print number of matches"
-msgstr ""
+msgstr "N’afficher que le nombre d’occurrences"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:10
-#, fuzzy
 msgid "Action for devices"
-msgstr "Afficher les dépendances importantes"
+msgstr "Action à effectuer sur les fichiers de périphérique"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:11
-#, fuzzy
 msgid "Action for directories"
-msgstr "Nettoyer les répertoires source"
+msgstr "Action à effectuer sur les répertoires"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:12
 msgid "Pattern is extended regexp"
-msgstr ""
+msgstr "Le motif est une expression régulière POSIX étendue"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:13
 msgid "Pattern is a regexp"
-msgstr ""
+msgstr "Le motif est une expression régulière"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:14
 msgid "Read pattern list from file. Skip files whose base name matches list"
 msgstr ""
+"Lire la liste des motifs depuis un fichier ; passer les fichiers dont le nom "
+"seul, sans chemin, apparaît dans la liste"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:15
 msgid "Exclude matching directories from recursive searches"
-msgstr ""
+msgstr "Exclure les répertoires correspondants des recherches récursives"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:16
 msgid "Pattern is a fixed string"
-msgstr ""
+msgstr "Le motif est une chaîne de caractères fixe"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:17
-#, fuzzy
 msgid "Use patterns from a file"
-msgstr "Lire le paquet depuis un fichier"
+msgstr "Lire les motifs depuis un fichier"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:18
 msgid "Pattern is basic regex"
-msgstr ""
+msgstr "Le motif est une expression régulière POSIX simple"
 
```

Comment:
Inconsistent terminology for basic regex: “expression régulière basique” above, now “expression régulière POSIX simple” (literally “simple POSIX regular expression”).